lib/src/posts.dart

```diff
@@ -595,6 +608,14 @@ class _PostActionsViewState extends State<PostActionsView> with StateLocalizatio
   }
 
   _showOriginalPost() => Navigator.pushNamed(context, "/post", arguments: widget.post.root);
+
+  _sharePost() {
+    Client client = context.read<Client>();
+    String hostname = client.currentUserId.split('@').last;
+    String guid = widget.post.guid;
```

Comment:
Type inference for the masses!

```suggestion
    final client = context.read<Client>(),
      hostname = client.currentUserId.split('@').last,
      guid = widget.post.guid;
```

Reply:
Did not know this formatting rule - until now..